### PR TITLE
Add simple instructions

### PR DESCRIPTION
**Pull request by ccarouge**
This is to add some simple instructions on how to use this as is so we can share more easily with people.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,3 +5,11 @@ An example payu experiment for running a spatial CABLE configuration (offline).
...
 
 **Note**: this experiment is still a work in progress and is dependent on the payu CABLE driver implementation (see https://github.com/payu-org/payu/pull/314)
+
+## Run the experiment
+
+1. Clone the CABLE repo: https://github.com/CABLE-LSM/CABLE
+2. In offline/build3.sh, change the MPI module to openmpi/4.1.4, then compile CABLE with MPI: `build3.sh mpi`
+3. Clone this cable_example repository
+4. Modify the config.yaml file with the path to your CABLE executable.
+5. Run with payu: `payu run`
```
Comment (SeanBryan51) [inline README.md]:
```suggestion
5. Run with payu: `payu run`. Note `payu` will need to be installed with CABLE driver (see https://github.com/payu-org/payu/pull/314)
```

File: config.yaml [hunk 2]
```diff
@@ -19,8 +19,6 @@ input:
     - /g/data/tm70/sb8430/cable/MetForcing/crujra_access/Wind
     - /g/data/tm70/sb8430/cable/MetForcing/crujra_access/crujra_access_landmask.nc
     - /g/data/tm70/sb8430/cable/MetForcing/crujra_access/crujra_access_gridinfo.nc
-    - /home/189/sb8430/cable/CABLE-AUX/core/biogeophys/def_veg_params_zr_clitt_albedo_fix.txt
-    - /home/189/sb8430/cable/CABLE-AUX/core/biogeophys/def_soil_params.txt
```
Comment (SeanBryan51) [inline config.yaml:23]: Why are we removing these inputs?

**Review by SeanBryan51 — CHANGES_REQUESTED** [review]

File: config.yaml (hunk 2 above)
Comment (ccarouge) [reply config.yaml:23]: These are only needed for CABLE v2. And I didn't want to confuse Paul with them when they aren't used for the main CABLE version.

Comment (SeanBryan51) [reply config.yaml:23]: Ok. Can you remove the entries in the namelist file? I.e. https://github.com/CABLE-LSM/cable_example/blob/51bf603f28bc53288515e81a9404f0d3ac889c91/cable.nml#L8-L9

**Review by SeanBryan51 — APPROVED** [review]
Happy to merge after the namelist file is updated.

**Pull request merged**